frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/review-and-create-step/review-and-create-step.tsx

```diff
@@ -0,0 +1,180 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  TextContent,
+  Text,
+  TextVariants,
+  Card,
+  CardBody,
+  AlertVariant,
+  Split,
+  SplitItem,
+} from '@patternfly/react-core';
+import {
+  GreenCheckCircleIcon,
+  YellowExclamationTriangleIcon,
+  BlueInfoCircleIcon,
+  RedExclamationCircleIcon,
+} from '@console/shared/src';
+import { humanizeBinaryBytes } from '@console/internal/components/utils';
+import { MINIMUM_NODES, NO_PROVISIONER } from '../../../../constants';
+import {
+  capacityAndNodesValidate,
+  getAllZone,
+  getTotalCpu,
+  getTotalMemory,
+} from '../../../../utils/create-storage-system';
+import { OSD_CAPACITY_SIZES, TotalCapacityText } from '../../../../utils/osd-size-dropdown';
+import {
+  ValidationMessage,
+  VALIDATIONS,
+  ValidationType,
+} from '../../../../utils/common-ocs-install-el';
+import { WizardNodeState, WizardState } from '../../reducer';
+import './review-and-create-step.scss';
+
+const NodesCard: React.FC<NodeCardProps> = ({ nodes }) =>
```

Comment:
We can reuse this card from `frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-wizard/review-and-create.tsx` , pass the classnames and improvise the state

Reply:
I am using this here since soon ocs one will be deprecated. We should not waste effort in maintaining older code.

Reply:
ack